家計簿アプリのエラー修正

@claude 

家計簿アプリの作成 #3 で作成したプログラムをPRでマージしたところ、以下のビルドエラーが発生しました。
こちらの修正をお願いします。

**Creating an optimized production build ...
[baseline-browser-mapping] The data in this module is over two months old.  To ensure accurate Baseline data, please update: `npm i baseline-browser-mapping@latest -D`
 ✓ Compiled successfully in 2.6s
   Running TypeScript ...
Failed to compile.

./app/components/HistoryTab.tsx:179:62
Type error: Expected 0-1 arguments, but got 2.

  177 |                     </div>
  178 |                     <div className="text-sm text-gray-600">
> 179 |                       {transaction.date.split('-').join('年', '月') + '日'} {transaction.description && `• ${transaction.description}`}
      |                                                              ^
  180 |                     </div>
  181 |                   </div>
  182 |                   <div className="flex gap-2">
Next.js build worker exited with code: 1 and signal: null
Error: Process completed with exit code 1.**

--- a/app/components/HistoryTab.tsx
+++ b/app/components/HistoryTab.tsx
@@ -176,7 +176,7 @@ export default function HistoryTab({
                       </span>
                     </div>
                     <div className="text-sm text-gray-600">
-                      {transaction.date.split('-').join('年', '月') + '日'} {transaction.description && `• ${transaction.description}`}
+                      {transaction.date.split('-')[0]}年{transaction.date.split('-')[1]}月{transaction.date.split('-')[2]}日 {transaction.description && `• ${transaction.description}`}
                     </div>
                   </div>
                   <div className="flex gap-2">

